### PR TITLE
Test cases and route changes for Updating a recommendation

### PR DESCRIPTION
**Pull request by rushab-shah**

### CODE REVIEW

File: service/models.py [hunk 1]
```diff
@@ -47,11 +47,13 @@ def create(self):
         db.session.add(self)
         db.session.commit()
 
-    def update(self):
+    def update(self,data):
```
Comment (yjlo123) [inline service/models.py:50]: should not add additional params to this method

File: tests/test_routes.py [hunk 2]
```diff
@@ -108,4 +108,90 @@ def test_get_not_found(self):
...
         resp = self.client.get(f"{BASE_URL}/0")
-        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
\ No newline at end of file
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+        
+    def test_update(self):
+        """It should update the given recommendation"""
+        pid = 100
+        rec = make_recommendation(pid,200,0)
+        resp = self.client.post(
```
Comment (yjlo123) [inline tests/test_routes.py:117]: should use `put` method to update

File: tests/test_routes.py [hunk 3]
```diff
@@ -108,4 +108,90 @@ def test_get_not_found(self):
         """It should not Read a Recommendation that is not found"""
         # Get a Recommendation that does not exist
         resp = self.client.get(f"{BASE_URL}/0")
-        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
\ No newline at end of file
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+        
+    def test_update(self):
```
Comment (yjlo123) [inline tests/test_routes.py:113]: suggesstion: can split this long test case into smaller ones

**Review by yjlo123 — CHANGES_REQUESTED** [review]

**Comment by rushab-shah** [comment]
@yjlo123 Changes made

**Pull request merged**